tests/netebpfext_unit/netebpfext_unit.cpp

```diff
@@ -7,6 +7,11 @@
 
 #include <map>
 
+#define SOCK_ADDR_TEST_ACTION_PERMIT 1
+#define SOCK_ADDR_TEST_ACTION_BLOCK 2
+#define SOCK_ADDR_TEST_ACTION_REDIRECT 3
+#define SOCK_ADDR_TEST_ACTION_FAILURE 4
```

Comment:
Can you change this to be an enum instead of using defines?  It makes debugging easier since the debugger can show the symbol instead of just the number.

Reply:
Fixed.